src/Tribe/Rewrite.php

```diff
@@ -317,6 +317,9 @@ public function get_bases( $method = 'regex' ) {
 		 */
 		$bases = apply_filters( 'tribe_events_rewrite_i18n_slugs_raw', $bases, $method, $domains );
 
+		// Again, make sure the bases are unique.
+		$bases = array_map( 'array_unique', $bases );
```

Comment:
This is good here, but I think you can remove it from above (line 286) - it's all filters except for `merge_localized_bases()` - which _also_ runs array_unique on `$bases`?

Not needed for this PR 😉 just an observation.

Reply:
It's actually required as filters can alter (and they do) the bases to add redundancy.